scripts/config-transformer.sh

```diff
@@ -33,6 +33,7 @@ mv $TEMPLATE_DIR/verticadbs.vertica.com-crd.yaml $CRD_DIR
 mv $TEMPLATE_DIR/verticaautoscalers.vertica.com-crd.yaml $CRD_DIR
 mv $TEMPLATE_DIR/eventtriggers.vertica.com-crd.yaml $CRD_DIR
 mv $TEMPLATE_DIR/verticarestorepointsqueries.vertica.com-crd.yaml $CRD_DIR
+mv $TEMPLATE_DIR/verticascrutinizes.vertica.com-crd.yaml $CRD_DIR
```

Comment:
I wonder if we can make this more future proof for new CRDs by using a wildcard?
```suggestion
mv $TEMPLATE_DIR/*-crd.yaml $CRD_DIR
```